### PR TITLE
OZ-24: Add a quick start script to Ozone FOSS.

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

**Review by rbuisson — CHANGES_REQUESTED** [review]
This fails when running on a Mac:
```
Failed to execute process './start-demo.sh'. Reason:
exec: Exec format error
The file './start-demo.sh' is marked as an executable but could not be run by the operating system.
```
Is the script specific to Linux architecture?

File: start-demo.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export OZONE_DIR=$PWD/ozone && \
```
Comment (rbuisson) [inline start-demo.sh:1]: There should be some shebang at the start of this file to inform how it must be interpreted.

File: start-demo.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+mkdir -p $OZONE_DIR
+# Download the project && \
+export VERSION=1.0.0-SNAPSHOT && \
+./mvnw org.apache.maven.plugins:maven-dependency-plugin:3.2.0:get -DremoteRepositories=https://nexus.mekomsolutions.net/repository/maven-public -Dartifact=net.mekomsolutions:ozone-distro:$VERSION:zip -Dtransitive=false --legacy-local-repository && \
+./mvnw org.apache.maven.plugins:maven-dependency-plugin:3.2.0:unpack -Dproject.basedir=$OZONE_DIR -Dartifact=net.mekomsolutions:ozone-distro:$VERSION:zip -DoutputDirectory=$OZONE_DIR/ozone-distro-$VERSION
+
+export DISTRO_PATH=$OZONE_DIR/ozone-distro-$VERSION;  \
```
Comment (rbuisson) [inline start-demo.sh:8]: Could this be made prettier?
There is no need for `;` and `\` for instance, since we are not copying it in a shell.

**Review by rbuisson — CHANGES_REQUESTED** [review]
Some clean up needed on the script itself.

Let's remember that this is the entrypoint for Ozone and we want this to look as pro as possible.

**Comment by jbravo94** [comment]
@enyachoke First of all thank you for this PR since it helps me in trying out ozone.

Currently i saw that the docker image `mekomsolutions/senaite:latest` is not publicly on DockerHub - therefore ozone could not start - just as a hint not to forget to make it public before merging, if somebody else wants to run the script.

I tried the old way by building the image via docker-compose, whereas i was only able to succeed by setting pyphen to 0.10.0 in requirements.txt (https://github.com/jbravo94/ozone-docker/blob/dcac289350e5cba956d97f50b55ca2c01ab4c298/senaite/requirements.txt#L4)

Maybe this is not the best file to enforce the version, but it worked for testing - i thought i mention it, maybe it helps you as well.

Furthermore, it looks that openmrs-initializer runs through, but afterwards openmrs backend gets unhealthy somehow and it returns me always 404 - at least at my PC also at SPA login: https://imgur.com/a/Xp6dsPZ

Can you tell me if openmrs backend works at your PC after running the script e.g. you can login via SPA?
Maybe i forgot something on my pc...

BR,
Johannes

**Comment by enyachoke** [comment]
Thanks @jbravo94 for trying this out   sorry about the SENAITE image.  We are aware of it , our CD is having issues building the image but we will get it published soon.

**Comment by enyachoke** [comment]
@jbravo94  for the OpenMRS 404 it is a know issue when using openmrs-initializer which has not been fixed. If you restart OpenMRS it will work Okay.

**Comment by jbravo94** [comment]
@enyachoke Thanks for the hint - bouncing the openmrs container worked.

As a workaround i made myself this snippet:
```
until $(docker ps --filter health=healthy | grep -q ozone-docker_openmrs_1); do sleep 2; done
until $(docker ps --filter health=unhealthy | grep -q ozone-docker_openmrs_1); do sleep 2; done
docker restart ozone-docker_openmrs_1 > /dev/null
```

**Review by rbuisson — APPROVED** [review]

**Pull request merged**